apps/web/src/components/VerificationCode/MethodButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { IUser, Void } from "@litespace/types";
+import { useFormatMessage } from "@litespace/ui/hooks/intl";
+import { LocalId } from "@litespace/ui/locales";
+import { Typography } from "@litespace/ui/Typography";
+import cn from "classnames";
+
+type MethodButtonProps = {
+  method: IUser.NotificationMethod;
+  isActive?: boolean;
+  onClick?: Void;
+  activeColor: string;
+  icon: React.ReactNode;
+};
+
+const MethodIntlMap: Record<IUser.NotificationMethod, LocalId> = {
+  [IUser.NotificationMethod.Telegram]: "notification.method.telegram",
+  [IUser.NotificationMethod.Whatsapp]: "notification.method.whatsapp",
+};
+
+export function MethodButton({
+  isActive,
+  icon,
+  method,
+  activeColor,
+  onClick,
+}: MethodButtonProps) {
+  const intl = useFormatMessage();
+
+  return (
+    <button
+      onClick={onClick}
+      type="button"
+      className={cn(
+        "flex grow flex-col items-center bg-natural-200 hover:bg-natural-300 justify-center gap-2 rounded-2xl transition-all duration-200",
+        isActive ? activeColor : ""
+      )}
+    >
+      {icon}
+      <Typography
+        tag="p"
+        className={cn(
+          "text-subtitle-1 font-medium",
+          isActive ? "text-white" : "text-natural-50"
```

Comment:
Why not using `natural-50` in all cases? I can see in the design that one card has `natural-50` but the other has white. Disucss it with Ziad please.